go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/xenserver/packer-builder-xenserver
 
-go 1.14
+go 1.16
```

Comment:
This is an unrelated change but since the GitHub release action is using go 1.16 I wanted to get this updated as well.